packages/react-components/components/BaseNotification.tsx

```diff
@@ -1,52 +1,95 @@
-import BaseListItem from '@celo/react-components/components/BaseListItem'
 import TextButton from '@celo/react-components/components/TextButton'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { elevationShadowStyle } from '@celo/react-components/styles/styles'
+import variables from '@celo/react-components/styles/variables'
 import * as React from 'react'
-import { StyleSheet, View } from 'react-native'
+import {
+  Platform,
+  StyleSheet,
+  Text,
+  TouchableNativeFeedback,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+
+const { contentPadding } = variables
 
 interface Props {
   icon?: React.ReactNode
   title: string
   children: React.ReactNode
-  callout?: React.ReactNode
   ctas: CTA[]
-  roundedBorders?: boolean
+  onPress?: () => unknown
 }
 
 export interface CTA {
   onPress: () => unknown
   text: string
 }
 
-// For use in Notification Center and Payment Request Screen
-export default function BaseNotification(props: Props) {
+function Wrapper({ onPress, children }: { children: React.ReactNode; onPress?: () => unknown }) {
+  return onPress ? (
+    Platform.OS === 'android' ? (
+      <TouchableNativeFeedback onPress={onPress}>{children}</TouchableNativeFeedback>
```

Comment:
Can you just use our custom `Touchable` component and make that work well for both iOS and Android?
packages/react-components/components/Touchable.tsx

Reply:
Oh, I sure can. I thought to make it more authentic for a user, so I've used `TouchableNativeFeedback`